modules/acquia_cms_headless/tests/src/Functional/HeadlessFrontpageTest.php

```diff
@@ -2,29 +2,36 @@
 
 namespace Drupal\Tests\acquia_cms_headless\Functional;
 
+use Drupal\Tests\BrowserTestBase;
+
 /**
  * Tests for acquia_cms_headless frontpage.
  *
  * @group acquia_cms_headless
  * @group low_risk
  */
-class HeadlessFrontpageTest extends HeadlessTestBase {
+class HeadlessFrontpageTest extends BrowserTestBase {
```

Comment:
@apathak18 any reason why this is changed to BrowserTestBase ?

cc @vishalkhode1

Reply:
@deepakmishra2 any reason why we extended HeadlessTestBase?